libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -6407,6 +6410,12 @@ bool GCS_MAVLINK::accept_packet(const mavlink_status_t &status,
         return true;
     }
 
+    // handle messages from a mavlink receiver
+    if ((msg.compid == MAV_COMP_ID_TELEMETRY_RADIO) &&
+        (msg.msgid == MAVLINK_MSG_ID_RADIO_RC_CHANNELS)) {
```

Comment:
this bypasses signing checks, for a packet that can control the vehicle, this is very dangerous, we must not bypass those checks

Reply:
I 100% agree that signing checks should not be bypassed, but I can't see anything in the function accept_packet() which would do something related to signing. 
Could you maybe drop a hint of what I'm missing?

Reply:
I'll need to look when not on my phone, I thought signing used this for a bypass check